setup.cfg

```diff
@@ -126,3 +126,10 @@ ignore = E203,W503,E501,C901
 max-line-length = 79
 max-complexity = 18
 exclude = _vendor,vendored,__init__.py,examples,benchmarks,napari/resources/_qt_resources*.py
+
+
+[coverage:report]
+exclude_lines =
+    pragma: no cover
```

Comment:
Just curious what is this/ what does it mean?

Reply:
that's the "ignore this line" command for [coverage.py](https://coverage.readthedocs.io/en/coverage-4.3.3/excluding.html) ... when you don't care if a given line is covered in test coverage

Reply:
thanks